vMenu/menus/VehicleOptions.cs

```diff
@@ -134,6 +134,21 @@ private void CreateMenu()
             };
             MenuListItem vehicleLights = new MenuListItem("Vehicle Lights", lights, 0, "Turn vehicle lights on/off.");
 
+            List<string> stationNames = new List<string>();
+            
+            foreach (var radioStationName in Enum.GetNames(typeof(RadioStation)))
+            {
+                stationNames.Add(radioStationName);
+            }
+
+            int radioIndex = UserDefaults.VehicleDefaultRadio;
+
+            if (radioIndex == 255) // 255 is radio off
+            {
+                radioIndex = Enum.GetValues(typeof(RadioStation)).Length -1; // the last item is Radio off
```

Comment:
Length can change, so this is not safe.
Try converting the enum into an int using `(int)RadioStation.Off` (or whatever the enum value is for the radio off) instead

Reply:
When i use `(int)RadioStation.RadioOff` i will just get 255 which is a value, i need index instead. I need an index to the default position of MenuListItem. I did some refactor so i don't rely on the Length 😄